README.md

```diff
@@ -1,23 +1,35 @@
-Role Name
+Ansible-kpi
 =========
 
-A brief description of the role goes here.
+Use this role to install and configure KPI.
 
-Requirements
+Role Dependencies
 ------------
 
-Any pre-requisites that may not be covered by Ansible itself or the role should
-be mentioned here. For instance, if the role uses the EC2 module, it may be a
-good idea to mention in this section that the boto package is required.
+- onaio.collectd
+- onaio.monit
+- onaio.postgresql
```

Comment:
Not a direct requirement. We only use it in the tests.

Reply:
oh...let me remove that